js/src/dom/event-handler.js

```diff
@@ -107,13 +107,13 @@ function bootstrapDelegationHandler(element, selector, fn) {
   return function handler(event) {
     const domElements = element.querySelectorAll(selector)
 
-    for (let { target } = event; target && target !== this; target = target.parentNode) {
+    for (let { target, type } = event; target && target !== this; target = target.parentNode) {
```

Comment:
This change is unneeded; there is no issue here.

Reply:
I would swear it was  a warning on my local, but as I cannot replicate I revert it

Reply:
Yeah, it happened in the past but it was an xo issue which is now fixed.